ajax/libs/spritejs/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "spritejs",
+  "version": "3.2.8",
+  "description": "A cross platform high-performance graphics system.",
+  "filename": "spritejs.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/spritejs/spritejs.git"
+  },
+  "author": "akira-cn",
+  "homepage": "http://spritejs.org",
+  "keywords": ["spritejs", "canvas", "graphics", "rendering", "render", "svg-path", "svg", "webgl2", "webgl", "3d", "2d"],
+  "license": "MIT",
+  "npmName": "spritejs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "spritejs.js",
```

Comment:
Are these the only files needed, not the `es` files?

```
├── dist
│   ├── spritejs.es.js
│   ├── spritejs.es.min.js
│   ├── spritejs.js
│   ├── spritejs.min.js
│   ├── spritejs.worker.es.js
│   ├── spritejs.worker.es.min.js
│   ├── spritejs.worker.js
│   └── spritejs.worker.min.js
```

Reply:
yes you're right, I should add es module file into it. And I asked author `spritejs.worker.js` is a bundle for web worker, it also should be added. Thank you for your figure out, I fixed it and please check again.